5.0/en/0x12-V3-Session-management.md

```diff
@@ -30,21 +30,19 @@ As previously noted, these requirements have been adapted to be a compliant subs
 
 TLS or another secure transport channel is mandatory for session management. This is covered in the Communications Security chapter.
 
-## V3.3 Session Termination
+## V3.3 Session Timeouts
```

Comment:
To align with the style of the rest of the titles, I think it should be "Session Timeout"